app/javascript/app/components/nav/nav-styles.scss

```diff
@@ -49,12 +49,29 @@
 
   &.active::after {
     height: 6px;
-    background-color: $white;
+    background-color: $butterscotch;
+  }
+}
+
+.stickyWrapper {
+  max-height: 34px;
+
+  :global {
+    .sticky-inner-wrapper {
+      background-color: $action-color;
```

Comment:
Not sure if this is the desired colour for the sticky navbar (there's no design for this state of the sub navigation).
Just pointing it out.

![image](https://user-images.githubusercontent.com/6906348/49787276-8e225780-fd26-11e8-9e42-18f6a75b1b97.png)

Reply:
well, actually, on the design the sub navigation isn't sticky. I will remove it then.